ChangeLog.md

```diff
@@ -1,3 +1,29 @@
+# ooniprobe-ios 1.1.3 [2017-03-08]
+
+Bugfixes:
+
+* Various GUI fixing
+
+* Expose app version number in app
+
+* Empty screen in "Past Tests" on init
+
+* Delete all tests button
+
+* Render failed "NDT Speed Test" 
+
+New languages:
+
+* IT
+
+* FR
+
+* ES
+
+measurement-kit version: 0.4.1
```

Comment:
Simone is going to do new release of MK so we should not forget to bump this once the dep is updated.